client/app/domain/tasks/TASKS.html

```diff
@@ -1,71 +1,35 @@
-                        </td>
-                        <td>
-                            <button class="btn btn-icon" type="button" title="{{i18n.wizard_modify}}"
-                                    data-ng-if="task.canCancel || task.canRelauch || task.canAccelerate"
-                                    data-ng-click="setAction('../domain-operation/update/domain-operation-update', task)">
-                                <span class="oui-icon oui-icon-pen_line oui-icon_small" aria-hidden="true"></span>
-                            </button>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-    </div>
-
-    <div class="clearfix" data-ng-if="ctrlDomainTasks.tasks">
-        <div data-pagination-front
-             data-items="ctrlDomainTasks.tasks"
-             data-paginated-items="ctrlDomainTasks.tasksDetails"
-             data-current-page="currentPage"
-             data-nb-pages="nbPages"
-             data-items-per-page="itemsPerPage"
-             data-transform-item="ctrlDomainTasks.transformItem(item)"
-             data-on-transform-item-get-promise="getPromise(promise)"
-             data-on-transform-item-notify="ctrlDomainTasks.onTransformItemNotify(item)"
-             data-on-transform-item-done="ctrlDomainTasks.onTransformItemDone(items)"
-             data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-             data-item-per-page-placeholder="{{tr('pagination_display')}}"
-             data-ng-show="!ctrlDomainTasks.loading && ctrlDomainTasks.tasksDetails.length > 0"></div>
-    </div>
+    <oui-datagrid data-rows="ctrlDomainTasks.tasks"
+        data-row-loader="ctrlDomainTasks.transformItem($row)">
+        <oui-column data-property="function"
+            data-title="tr('domain_tab_TASKS_function')"></oui-column>
+        <oui-column data-property="todoDate"
+            data-title="tr('domain_tab_TASKS_function')">
+            {{ $row.todoDate | date:'mediumDate' }}
+        </oui-column>
+        <oui-column data-property="comment"
+            data-title="tr('domain_tab_TASKS_comment')">
+            {{ $row.comment || '-' }}
+        </oui-column>
+        <oui-column property="status"
+            data-title="tr('domain_tab_TASKS_state')"
+            class="text-center">
+            <span class="label"
+                  data-ng-class="{
+                    'label-info': $row.status === 'TODO' || $row.status === 'DOING',
+                    'label-danger': $row.status === 'ERROR',
+                    'label-default': $row.status === 'CANCELLED',
+                    'label-success': $row.status === 'DONE'
+                  }"
+                  data-ng-bind="tr('domain_tab_TASKS_status_' +  $row.status)">
+            </span>
+        </oui-column>
+        <oui-action-menu data-ng-if="$row.canCancel || $row.canRelauch || $row.canAccelerate"
+            data-align="end"
+            data-compact>
```

Comment:
What is a compact button ? equivalent of xs in bootstrap ? I can't find a doc :( except "Use a compact button" ^^

Reply:
Nope, it's the rounded button for action-menus. If the action-menu does not specifies this attribute, the button looks like a regular dropdown.